cpp/src/io/json/json_normalization.cu

```diff
@@ -298,52 +298,54 @@ struct TransduceToNormalizedWS {
 
 namespace detail {
 
-rmm::device_uvector<SymbolT> normalize_single_quotes(rmm::device_uvector<SymbolT>&& inbuf,
-                                                     rmm::cuda_stream_view stream,
-                                                     rmm::device_async_resource_ref mr)
+void normalize_single_quotes(datasource::owning_buffer<rmm::device_uvector<char>>& indata,
```

Comment:
Trivial nitpick: We've changed the parameter name to `indata`.  Do we want to keep this name consistent with the header?

Reply:
Yes, I agree that the naming should be consistent. Fixed!